plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/ElasticsearchQueryRunner.java

```diff
@@ -124,16 +127,22 @@ public static Session createSession()
         return testSessionBuilder().setCatalog("elasticsearch").setSchema(TPCH_SCHEMA).build();
     }
 
+    private static HostAndPort getAddress()
+            throws IOException
+    {
+        if (START_ELASTICSEARCH_SERVER) {
+            return new ElasticsearchServer("docker.elastic.co/elasticsearch/elasticsearch:7.6.2", ImmutableMap.of()).getAddress();
```

Comment:
How is the server lifecycle managed? How does it get stopped?

Reply:
The underlying container would be managed by Testcontainers ryuk. It will be closed with some delay when stopping the query runner.

Reply:
We have some rudimentary support for "container reuse", which i think it what you're looking for.

I propose you use that, and let's avoid adding a yet new method of skipping container creation.

BTW, for manual testing, I find PT envs very useful. `ptl env up .. --without-trino` + `DevelopmentServer` is what i use locally. This requires no changes in the query runners. Of course, this works for manual testing, and doesn't work for running integration tests (those using QueryRunner), but i understand this is the use-case you're addressing here.



cc @kokosing @losipiuk

Reply:
@martint i see your point, but that's not how QueryRunners.main are generally implemented.
They are self-contained first and foremost, and then re-usable if needed. I don't think ES should be special here.

Reply:
How about adding another query runner for external service usage? Then, we can use `ElasticsearchQueryRunner` for dockerized environment.
```java
public final class ElasticsearchExternalQueryRunner
{
    private ElasticsearchExternalQueryRunner() {}

    public static void main(String[] args)
            throws Exception
    {
        Logging.initialize();

        DistributedQueryRunner queryRunner = createElasticsearchQueryRunner(
                HostAndPort.fromParts("localhost", 9200),
                TpchTable.getTables(),
                ImmutableMap.of("http-server.http.port", "8080"),
                ImmutableMap.of(),
                3);

        Logger log = Logger.get(ElasticsearchExternalQueryRunner.class);
        log.info("======== SERVER STARTED ========");
        log.info("\n====\n%s\n====", queryRunner.getCoordinator().getBaseUrl());
    }
}
```

Reply:
@ebyhr that sounds like a reasonable compromise.
in the `ElasticsearchExternalQueryRunner` please include a comment intructing hwo to start this external thing

Reply:
@findepi @martint Added another query runner.